### ISSUE TITLE
Release build failing

### ISSUE THREAD

**Issue by jacques-n** [substrait-io/substrait-java#108]
I merged the release patch it looks like the release build is failing.

Additionally, it looks like it is tagging version on each merge (maybe) as opposed to once a week...

Failure here: https://github.com/substrait-io/substrait-java/actions/runs/3544635664/jobs/5952198598

@davisusanibar, can you take a look?

**Comment by davisusanibar** [comment]
Let me check that

**Comment by davisusanibar** [comment]
Just to mention:
- Building error: Solved by upgrade `gradlew` version.
- Tagging version: It was implemented to deploy immediately after a PR is merged (Is it necessary to wait/concatenate features to release then?). Changed to deploy `2 AM on Sunday` or manually.

For testing purpose, please help me to run release job to test if it is working, thanks.

**Comment by davisusanibar** [comment]
Hi @jacques-n / @westonpace 

I'am seeing this error message on the release: https://github.com/substrait-io/substrait-java/actions/runs/3550260556/jobs/5963588183
````
* What went wrong:
Could not evaluate onlyIf predicate for task ':core:signMaven-publishPublication'.
> Could not read PGP secret key
````

Could you try to run `Sanity Check Environment` to test secrets configuration?

**Comment by jacques-n** [comment]
Reopening as the release is still failing.

**Comment by jacques-n** [comment]
@davisusanibar : https://github.com/substrait-io/substrait-java/actions/runs/3550853020/jobs/5964589379

**Comment by davisusanibar** [comment]
Please update secret value for `SIGNING_KEY`

`SIGNING_KEY` needs to be generated with this command line: `gpg --export-secret-keys C8BA52B3 | base64`
 
As the message mention `base64: invalid input` I could suggest that the current value is invalid, please in case this was created with `gpg --export-secret-keys --armor C8BA52B3`, please change to use base64 command line mentioned.

@westonpace / @jacques-n

**Comment by westonpace** [comment]
Yes, I had exported the key in `--armor` format (I think this is what Sonatype wants).  Either way, I have updated they key to be base64 encoded non-armor.  The sanity script now passes.  The release script failed because the signing key was invalid.  It wanted an 8-digit signing key.  I had included the full signing key.  I have changed the signing key secret to the short key.

This is ready to run again.

However, semantic release refuses to create a release because it doesn't see any changes.

So we will either need to create a dummy commit or wait for a change.

**Comment by davisusanibar** [comment]
An option could be to review https://github.com/substrait-io/substrait-java/pull/104 that is related with promote version from Staging to Production after 14 days old, and use that to test semantic-release process.

**Comment by davisusanibar** [comment]
Release available at:
- Core: https://search.maven.org/artifact/io.substrait/core/0.1.4/jar
- Isthmus: https://search.maven.org/artifact/io.substrait/isthmus/0.1.4/jar

**Comment by nielspardon** [comment]
The release build has been working reliably for a while now. I'm closing this issue and would suggest to open a new issue should the build fail in the future.
